Item edit destroy

aki800 opened the pull request:
#WHAT
商品の編集機能を実装

#WHY
商品を編集するため

商品を編集する様子↓
https://gyazo.com/6553630fb4c94e97a5c698fe313732dc

Commit:
Merge remote-tracking branch 'noncover222kdy/master' into item-edit-destroy

Commit:
編集機能の実装

#WHAT
商品の編集機能を実装

#WHY
商品を編集するため

TOMARUMARU commented on app/controllers/items_controller.rb:
```diff
@@ -28,6 +28,12 @@ def show
   end
 
   def update
+    item = Item.find(params[:id])
```
editやshowでも使用しているので、メソッドに切り分けてbefore_actionで呼び出しましょう

Commit:
items_controllerの修正

#WHAT
item = Item.find(params[:id])が
複数のアクションで使われるので、メソッドを定義して共通化

#WHY
効率的なコードを書くため。

TOMARUMARU reviewed (approved):
LGTM
